cypress/e2e/app.cy.ts

```diff
@@ -231,6 +231,41 @@ describe('/mock', () => {
     }
   });
 
+  it('search items in the file', () => {
+    cy.findByRole('tab', { name: 'Open search panel' })
+      .click()
+      .should('have.attr', 'aria-selected', 'true');
+
+    cy.findByRole('textbox', { name: 'Path to search' }).type('scatter');
+
+    cy.findAllByRole('treeitem')
+      .should('have.length', 2)
+      .each((e) => expect(e).to.contain('scatter'));
+  });
+
+  it.only('visualize search entry', () => {
+    cy.findByRole('tab', { name: 'Open search panel' }).click();
+
+    cy.findByRole('textbox', { name: 'Path to search' }).type('scatter');
+
+    cy.findByRole('treeitem', { name: '/nexus_entry/scatter' }).click();
+    cy.waitForStableDOM();
+
+    cy.findByRole('tab', { name: 'NX Scatter' }).should(...BE_SELECTED);
+    cy.findByRole('figure', { name: 'scatter_data' }).should('be.visible');
+
+    // Check that the selected node is kept when tabbing in Explorer
+    cy.findByRole('tab', { name: 'Open explorer panel' }).click();
+    cy.findExplorerNode('scatter').should('have.attr', 'aria-selected', 'true');
+
+    // Check that the search value is kept when tabbing back from Explorer
+    cy.findByRole('tab', { name: 'Open search panel' }).click();
+    cy.findByRole('textbox', { name: 'Path to search' }).should(
+      'have.value',
+      'scatter'
+    );
+  });
```

Comment:
These two tests could be merged to avoid duplicating the early steps. 

But I thought that the resulting test would be too long.

Reply:
Since this is E2E, I wouldn't see a problem with merging the two. I wouldn't bother expecting for the search and `NX Scatter` tabs to be selected, though.

You could maybe add an expectation that the `scatter` node receives focus when tabbing back to the explorer.